.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "extends": ["airbnb", "prettier"],
+  "extends": ["airbnb", "prettier", "plugin:grommet/recommended"],
   "plugins": ["prettier"],
```

Comment:
So this accesses the `recommended` property of the module config from `lib/index.js`?

Is the reason it gets placed in `extends` vs. `plugins` is it is auto applying? Whereas in `plugins` you are opting in a-la-carte?

Reply:
Should this be added to the [README.md](https://github.com/grommet/eslint-plugin-grommet/blob/master/README.md)?

Reply:
Good call, I'll add some guidance in the README explaining this and how to use.